tests_dir/logic/test_registeration_logic.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from myapp import models
+from myapp.logic import register_logic
+
+
+class TestRegisterLogic(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user(username='testuser', password='12345')
+        login = self.client.login(username='testuser', password='12345')
+
+    def test_create_profile(self):
```

Comment:
what if the same user registers again?

Reply:
Django will prevent that due to distinct key constraint not met